chest.lua

```diff
@@ -11,35 +11,69 @@ local function get_item_count(pos, player, count)
 	return count
 end
 
+-- Just in case we are under MineClone
+local default = rawget(_G, "default") or nil
+if default == nil then
+	default = rawget(_G, "mcl_sounds")
+end
+
 local tex_mod = "^[colorize:#FF2:50"
 minetest.register_node("simple_protection:chest", {
 	description = S("Shared Chest") .. " " .. S("(by protection)"),
 	tiles = {
-		"default_chest_top.png"  .. tex_mod,
-		"default_chest_top.png"  .. tex_mod,
-		"default_chest_side.png" .. tex_mod,
-		"default_chest_side.png" .. tex_mod,
-		"default_chest_side.png" .. tex_mod,
-		"default_chest_lock.png" .. tex_mod
+		"simple_protection_chest_top.png"  .. tex_mod,
+		"simple_protection_chest_top.png"  .. tex_mod,
+		"simple_protection_chest_side.png" .. tex_mod,
+		"simple_protection_chest_side.png" .. tex_mod,
+		"simple_protection_chest_side.png" .. tex_mod,
+		"simple_protection_chest_lock.png" .. tex_mod
 	},
 	paramtype2 = "facedir",
 	sounds = default.node_sound_wood_defaults(),
-	groups = {choppy = 2, oddly_breakable_by_hand = 2},
+	groups = {
+		-- Minetest Game
+		choppy = 2, oddly_breakable_by_hand = 2,
+		-- MineClone
+		handy=1,axey=1, deco_block=1,
```

Comment:
Should be moved to a variable which changes depending on the game.

Reply:
Technically speaking it can be left here, since Minetest Game uses different variables/groups than MineClone so there is no collision, but if you insist on it being a var okay. (I don't care so long as it works)

Reply:
Right, I guess it does not matter too much for node groups. In the worst case, a group could be misinterpreted as another functionality/property that is not intended by one of the games.

Reply:
Anyway I've move this also to a variable changing it depending on game.